distributions/standalone/src/main/resources/conf/scheduler-pools.conf

```diff
@@ -0,0 +1,27 @@
+# Configures the thread pools of the Mule Runtime
```

Comment:
IMO we should use a  mule specific prefix for these properties and also support allowing them to be configured via system properties.

e.g. `org.mule.runtime.scheduler.io.core_threads`